frontend/src/views/CollabPageView.tsx

```diff
@@ -79,6 +94,26 @@ function CollabPageView() {
 		}
 	};
 
+	// Callback function to mount editor to auto-focus when page loads
+	const onMount = (editor) => {
```

Comment:
I think this will throw an error if you rebuild your docker compose since editor is not defined. Consider putting an `any` type even though its bad practice... Alternatively can consider looking up the file type before putting it in and seeing if it works

```typescript
declare const _default: react.MemoExoticComponent<typeof Editor>;
```

Reply:
noted ! changed to use the monaco-specific type
```
// Callback function to mount editor to auto-focus when page loads
const onMount = (editor: monaco.editor.IStandaloneCodeEditor) => {
editorRef.current = editor;
editor.focus();
};
```

tested docker compose with `docker-compose up -d --no-cache` and its running fine 👍 
<br>
![image](https://github.com/user-attachments/assets/877100ea-c2d0-4860-84ee-2e9e0bba1efe)
![image](https://github.com/user-attachments/assets/63b57a69-f1c9-4c95-9a42-d6cedc76d24e)